packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -625,9 +609,6 @@ const VirtualListBaseFactory = (type) => {
 						node.blur();
 					}
 				}
-				if (type === JS) {
```

Comment:
We still need these code. Please do not remove them.

Reply:
Revived. Sorry. :)